tests/test_collect_peak_pairs.py

```diff
@@ -26,3 +28,29 @@ def test_cosine_hungarian_tolerance_01_shift_min5():
     matching_pairs = collect_peak_pairs(spec1, spec2, tolerance=0.2, shift=-5.0)
     assert len(matching_pairs) == 2, "Expected different number of matching peaks"
     assert matching_pairs == [(0, 0, 1.0), (1, 1, 1.0)], "Expected different matchin pairs."
+
+
+def test_cosine_hungarian_tolerance_01():
```

Comment:
Use https://docs.pytest.org/en/stable/parametrize.html to not duplicate tests

Reply:
I added this, but still kept 2 (instead of 4 before) tests, one for a compiled version and one for  a pure python version.
I could probably also call ``.py_func`` somehow via ``@pytest.mark.parametrize`` but wasn't sure how to do this in a clean way (without spending too much time on it).

Reply:
ok